### ISSUE TITLE
Anonymous Auth

### ISSUE THREAD

**Issue by lingrlongr** [mathcamp/pypicloud#62]
When configured to use ldap auth, anonymous uses browsing the pypi site cannot see any packages.  Upon logging in with ldap auth, you can see the packages.

If you disabled ldap auth in the config file, you can see the packages as an anonymous user.

I have this set in the config:
```
    pypi.default_read =
        everyone
    pypi.default_write =
        authenticated
```

Am I missing a configuration?

**Comment by stevearc** [comment]
I think there was a bug with the LDAP configuration. Could you try out this branch and let me know if it fixes your issue? https://github.com/mathcamp/pypicloud/tree/0.3

**Comment by lingrlongr** [comment]
That worked! Thanks for the quick fix!

**Comment by stevearc** [comment]
Fixed and released in [0.3.6](https://pypi.python.org/pypi/pypicloud/0.3.6)
